tox.ini

```diff
@@ -37,3 +37,9 @@ commands =
   coverage3 run --source prototype_template --parallel-mode -m pytest --doctest-modules
   coverage3 combine
   coverage3 report --fail-under=80
+
+[testenv:docs]
+deps =
+    -r{toxinidir}/requirements-dev.txt
```

Comment:
Are you certain that `requirements.txt` should never be required to build the docs?  I wouldn't be entirely surprised if there were some case in which it were necessary.

On the other hand, qiskit seems to do it like you did, so seems like a reasonable default for now.

Also, I'll just note that qiskit packages typically require `sphinx-intl` here as well.  We probably don't need this since we don't expect to have translations.